### ISSUE TITLE
Prevent default handling of passive touch events in Chrome

### ISSUE THREAD

**Issue by matze09** [mosch/react-avatar-editor#250]
### Behaviour
When trying to move the avatar (change x/y position with touch), the page scrolls instead. This only happens on Chrome on touch devices.

### Steps to reproduce
* Open [demo page](https://react-avatar-editor.netlify.com) in Mobile Chrome or Chrome with responsive mode activated in dev tools.
* Zoom in the avatar a bit, then try to move the avatar around
* Instead of the avatar being moved, the page scrolls (if it is low/narrow enough to require scrolling)
* In the Browser console, you can see a hint from Chrome saying: 
```
[Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/features/5093566007214080
```

### Background
Since Chrome 56, the `touchstart` and `touchmove` events are "passive" by default [[1]](https://www.chromestatus.com/features/5093566007214080). This means that avoiding the default handling (= browser scrolls page) of the event by setting `event.preventDefault()` does not work anymore, unless the event is explicitly configured as being active.
This has apparently been done by the Chrome devs in order to improve the scrolling performance on mobile devices [[2]](https://developers.google.com/web/updates/2017/01/scrolling-intervention#background_cancelable_events_slow_your_page_down).

### Fix
The recommended - and easy - way of fixing this is to just set the CSS property `touch-action: none` to the avatar element. This avoids all default browser panning and zooming within that element [[3]](https://developers.google.com/web/updates/2017/01/scrolling-intervention#breakage_and_guidance). This is also common for elements which bring their own panning/zooming behaviour, which I think this avatar is (cf. "Examples" [[4]](https://developer.mozilla.org/en-US/docs/Web/CSS/touch-action)).

A quick test setting `touch-action: none` fixed this issue on Chrome. It seems that Safari does not support this property though [[5]](https://bugs.webkit.org/show_bug.cgi?id=133112), but then it might not have the initial issue of passive events in the first place.

I will prepare a branch with this fix applied and do some further testing.
Comments and opinions welcome :-)

**Comment by matze09** [comment]
@mosch Thanks a lot for the quick review, merge and release!

**Comment by mikethejet** [comment]
Any news on that thanks you?

**Comment by joannajerz** [comment]
Hi, this problem is still on the chrome with touch

**Comment by mosch** [comment]
This was released in v11.0.3, what Version are you running?

**Comment by hemantkumbhar10** [comment]
In my case CSS didn't work. So I solved it by using ref itself

`const canvasref = editorRef.current.getImage();
 canvasref.style.touchAction='none';`
